scala/data_structures/bloom_filter.scala

```diff
@@ -0,0 +1,55 @@
+package data_structures
+
+import scala.util.hashing.MurmurHash3
+import scala.util.Random
+
+case class BloomFilter[A](
+  nbOfItems: Int,
+  falsePositiveProbability: Double,
+  private val array: Array[Boolean],
+  private val hashFunctions: List[String => Int]
+) {
+
+  def +=(elem: A): BloomFilter[A] = {
+    val str = elem.toString
+    val hashes = this.hashFunctions.map(_(str))
+    val newArr = hashes.foldLeft(this.array)((acc, hash) => acc.updated(hash, true))
```

Comment:
would avoid abbreviation in general

Reply:
idk what u mean

Reply:
for the variable names

Reply:
I was making a joke :)
I'll make the change

Reply:
that code will hit production no time for jokes @Bertrand31

Reply:
:fearful: